### ISSUE TITLE
Compilation error

### ISSUE THREAD

**Issue by finchy** [abates/pcap2flow#2]
```
$ ./build.sh 
pcap2flow.c:19:10: fatal error: pcap/pcap.h: No such file or directory
   19 | #include <pcap/pcap.h>
      |          ^~~~~~~~~~~~~
compilation terminated.
```

**Comment by finchy** [comment]
`sudo apt-get install libpcap-dev` on ubuntu fixes the above.. but then you run into:

```
pcap2flow.c: In function ‘main’:
pcap2flow.c:131:36: warning: implicit declaration of function ‘atoi’ [-Wimplicit-function-declaration]
  131 |   nf_init_peer(&nf_peer, &peer_ip, atoi(argv[3]));
      |    
```

**Comment by finchy** [comment]
Adding `#include <stdlib.h>` to `pcap2flow.c` fixes comment#2
